src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs

```diff
@@ -384,7 +384,7 @@ impl HintProcessor for BuiltinHintProcessor {
                     constants,
                 )
             }
-            hint_code::BLOCK_PERMUTATION => {
+            hint_code::BLOCK_PERMUTATION | hint_code::BLOCK_PERMUTATION_WHITELIST => {
```

Comment:
Are there any tests matching the new case?

Reply:
Added a parallel to _keccak.cairo test file with this changed hint (521fd15)